README.md

```diff
@@ -189,6 +190,23 @@ as a pre-commit.
 
 **NOTE**: the `protect` command can only be used on git repos, running `protect` on files or directories will result in an error message.
 
+### Creating a baseline
+
+When scanning large repositories or repositories with a long history, it can be convenient to use a baseline. When using a baseline, 
+gitleaks will ignore any old findings which is present in the baseline. To create a baseline, run gitleaks with the `--report-path` parameter. 
+
+```
+gitleaks detect --report-path .gitleaksbaseline # This will save the report in a file called .gitleaksbaseline
```

Comment:
I think we should rename the report file to something more conventional like `gitleaks-report.json`. Naming it `.gitleaksbaseline` might give readers the misconception that `--report-path` is something other than a gitleaks report.